### PR TITLE
Yet more CLI dev

### PR DESCRIPTION
**Pull request by dcoutts**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ Some of the more important settings are as follows:
 
 ## Logging
 
-Logs are output to the `log/` dir.
+Logs are output to the `log/s` dir.
```
Comment (ArturWieczorek) [inline README.md:106]: `logs` instead of `log/s` ?

Comment (dcoutts) [reply README.md:106]: yeah, that's odd

Comment (erikd) [reply README.md]: I have dropped that commit from this PR.

Comment (erikd) [reply README.md]: I have re-instated that commit, without my screwup :D

**Pull request merged**